sqlite/test/general/strictMode.test.js

```diff
@@ -0,0 +1,117 @@
+const cds = require('../../../test/cds.js')
+process.env.cds_features_db__strict = 'true'
+
+describe('strict mode', () => {
+  cds.test(__dirname, 'model.cds')
+
+  afterAll(() => {
+    process.env.cds_features_db__strict = undefined
+  })
```

Comment:
This should fix the issue that it would impact other tests. If tests impact other tests it is usually a flaw in the test definition.

```suggestion
describe('strict mode', () => {
  cds.test(__dirname, 'model.cds')

  beforeAll(() => {
    process.env.cds_features_db__strict = 'true'
  })

  afterAll(() => {
    process.env.cds_features_db__strict = undefined
  })
```

Reply:
i already tried this solution. But still the env variable was true in the other tests